apps/annotation/opencv_annotation.cpp

```diff
@@ -107,42 +110,33 @@ void on_mouse(int event, int x, int y, int , void * )
     }
 }
 
-// FUNCTION : snippet to convert an integer value to a string using a clean function
-// instead of creating a stringstream each time inside the main code
-string int2string(int num)
-{
-    stringstream temp_stream;
-    temp_stream << num;
-    return temp_stream.str();
-}
-
 // FUNCTION : given an image containing positive object instances, add all the object
 // annotations to a known stringstream
-void get_annotations(Mat input_image, stringstream* output_stream)
+void get_annotations(Mat input_image, vector<Rect>* current_annotations)
```

Comment:
@mshabunin I will update both master and 2.4 PR with these suggested updates and push it back!